criu/log.c

```diff
@@ -351,7 +350,7 @@ static void early_vprint(const char *format, unsigned int loglevel, va_list para
 	early_log_buf_off += log_size;
 }
 
-static void vprint_on_level(unsigned int loglevel, const char *format, va_list params)
+void vprint_on_level(unsigned int loglevel, const char *format, va_list params)
```

Comment:
I think this change should have been done in the third patch, otherwise it would break the build and bisect

Reply:
@prakritigoyal19 , is it done in the new version of the pr?

Reply:
Yes sir, I have moved this change to patch 3 - flog: add option to fallback to default textual logging